### ISSUE TITLE
Tunnel Not Found

### ISSUE THREAD

**Issue by mojaas** [htr-tech/zphisher#218]
Only wokrs on LAN not over the internet

**Comment by adan984** [comment]
i also have the same issue it is the majorf issue in zphisher

**Comment by mojaas** [comment]
> i also have the same issue it is the majorf issue in zphisher

Update me if u fixed it

**Comment by adan984** [comment]
NO I HAVE LOOKIN FOR CREATER HTR TO COMPLETE THIS MAJOR ISSUE PLEASE SOLVE THIS

**Comment by adan984** [comment]
PLEASE HELP US PLEASE

**Comment by htr-tech** [comment]
are you guyz creating link with ngrok ? 😕

**Comment by adan984** [comment]
yes we are creating with ngrok and when we sent to vicitim our change our network the link does not work and shows error tunnel not  found

**Comment by adan984** [comment]
blob:https://web.whatsapp.com/a5ac83c3-bf52-4ceb-ba18-961adbf620b1

**Comment by yudi2121** [comment]
ngrok since 2021 can.'t make tunnel for free

**Comment by adan984** [comment]
so can you tell us how can we get  ngrok paid sserv8ice pleasetell al the producer

**Comment by abass65** [comment]
Whats going on


Use termux

**Comment by abass65** [comment]
Go to google .next type ngrok.com and sing up next the lagest link in the
page copy  and past termux

**Comment by adan984** [comment]
bro we are apple user and using  ubuntun kali lionux please help

**Comment by B4LL00N** [comment]
make sure to port forward smartie Port:8080
